cmake/Modules/FindSystemBLAS.cmake

```diff
@@ -53,10 +67,6 @@ else()
     )
   endif()
 
-  if(BLAS_FOUND)
-    set(SystemBLAS_FOUND TRUE)
```

Comment:
I see you're not setting `SystemBLAS_FOUND` anymore which is fair. I suggest that you also remove the 2 other occurences of `SystemBLAS_FOUND` in this file. Looking at [find_package_handle_standard_args](https://cmake.org/cmake/help/latest/module/FindPackageHandleStandardArgs.html) `FOUND_VAR` should not be needed anymore and can be removed for our minimum CMake version.

Reply:
yes, I was still experimenting with it and, for what I saw, it isn't used anywhere so I am going to remove it.